### DISCUSSION TITLE
i have question about setCalendarVisibility behavior.

### DISCUSSION THREAD

**Discussion by wbh3110** [nhn/tui.calendar#1314] [Q&A]
Hello, I'm developing with vue.js 2.x

Can't remember the value of 'setCalendarVisibility'
I want invisible specific calendar when call 'clear' or 'createEvents'

Likewise... tutorial-calendar-app.
when the range is changed, you can see that the events of the hidden calendar are re-rendered.
Is this the intended behavior?

**Comment by adhrinae** [comment]
I think it's a bug. I need to investigate this problem and will create an issue about this.
